clear-intermediate-datasets/python-runnables/clear_int_datasets/runnable.py

```diff
@@ -66,13 +66,15 @@ def run(self, progress_callback):
             rcp_outputs_dict = rcp["outputs"]
             # CASE: no input dataset
             if rcp_inputs_dict:
-                input_key = list(rcp_inputs_dict.keys())[0]
-                rcp_inputs_list = [x["ref"] for x in rcp_inputs_dict[input_key]["items"]]
-                input_datasets += rcp_inputs_list
-            output_key = list(rcp_outputs_dict.keys())[0]
-            rcp_outputs_list = [x["ref"] for x in rcp_outputs_dict[output_key]["items"]]
-            # Append them to the overall input list:
-            output_datasets += rcp_outputs_list
+                input_keys = list(rcp_inputs_dict.keys())
+                for input_key in input_keys:
+                    rcp_inputs_list = [x["ref"] for x in rcp_inputs_dict[input_key]["items"]]
```

Comment:
rcp_inputs_list can be removed, by doing
 `input_datasets += [x["ref"] for x in rcp_inputs_dict[input_key]["items"]]`

Reply:
Done on commit [9bd8c34e934ba48fea3ae179da8bfb02d58d7cdb](https://github.com/gbetegon88/dataiku-contrib/commit/9bd8c34e934ba48fea3ae179da8bfb02d58d7cdb)